science/HDF5-External-Filter-Plugins/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           muniversal 1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
+
+github.setup        nexusformat HDF5-External-Filter-Plugins 0.1.0 v
+categories          science
+platforms           darwin
+license             BSD MIT
+maintainers         {@eborisch eborisch} openmaintainer
+description         Provides dynamically loadable (compression) filters for HDF5
+long_description    ${description}. Filters for bitshuffle, bzip2, and lz4 are \
+                    included. See also \
+                    http://www.hdfgroup.org/services/filters.html for \
+                    more information.
```

Comment:
Maybe just set the `homepage` to the above link and remove it from the long_description entirely.

Reply:
This URL is more of general information for all HDF5 filters, while the (github) homepage is for this particular set of filters. Planning to leave this way.